src/renderer/ModalsLayer.js

```diff
@@ -28,30 +28,39 @@ const BackDrop: ThemedComponent<{ state: string }> = styled.div.attrs(({ state }
   transition: opacity 200ms cubic-bezier(0.3, 1, 0.5, 0.8);
 `;
 
-const ModalsLayer = ({ visibleModals }: *) => (
-  <Transition
-    in={visibleModals.length > 0}
-    appear
-    mountOnEnter
-    unmountOnExit
-    timeout={{
-      appear: 100,
-      enter: 100,
-      exit: 200,
-    }}
-  >
-    {state => (
-      <BackDrop state={state}>
-        {/* {// Will only render at the end of december
+const ModalsLayer = ({ visibleModals }: *) => {
+  const filteredModals = visibleModals.filter(
+    ({ name, MODAL_SHOW_ONCE }) => !MODAL_SHOW_ONCE || !global.sessionStorage.getItem(name),
```

Comment:
(minor) usage of THIS_STYLE_OF_VAR_NAME is weird, we should only do that for constants, here it's more some kind of props, no?